tools/accuracy_checker/tests/test_adapters.py

```diff
@@ -17,8 +17,8 @@
 import numpy as np
 import pytest
 
-from openvino.tools.accuracy_checker.adapters import SSDAdapter, Adapter
-from openvino.tools.accuracy_checker.config import ConfigError
+from accuracy_checker.adapters import SSDAdapter, Adapter
```

Comment:
is validation team aware of these changes? I mean do they need to update some internal code which imported AC via `openvino.tools` ? or they use only AC as a CMD tool?

Reply:
I have no idea) I opened https://github.com/openvinotoolkit/openvino/pull/22683 to partially check this by running precommit. @eaidova, whom should I notify?

Reply:
@pwolnows @sstrehlk  please be aware about these changes in accuracy checker. It may requires to update imports in  accuracy_check,py in jenckins ci scripts, and injection annotation_converters_ext